packages/x-date-pickers/src/PickersActionBar/PickersActionBar.tsx

```diff
@@ -20,6 +21,12 @@ export interface PickersActionBarProps extends DialogActionsProps {
   onSetToday: () => void;
 }
 
+const PickersActionBarRoot = styled(DialogActions, {
```

Comment:
@LukasTy I don't know if it shouldbe `MuiPickersLayout.ActionBar` or `MuiPickersActionBar.Root`

I suspect that, once we develop a Base UI DX for the structure of the picker, it will all be part of the same `Picker` component.

It would look something like this:

```tsx
<Picker.Root>
  <DateField />
  <Picker.Modal> // The DX for the modal is very unclear to me right now
    <Picker.LayoutContainer>
      <DateCalendar />
      <Picker.ActionBar>
        <Picker.Action action="submit" />
      </Picker.ActionBar>
    <Picker.LayoutContainer>
  </Picker.Modal>
</Picker.Root>
```

In which case keeping the action bar inside the layout from a `styleOverrides` perspective makes sense.